ckcp/openshift_dev_setup.sh

```diff
@@ -301,12 +302,26 @@ install_openshfit_pipeline() {
   echo -n "  - openshift-pipeline application: "
   kubectl apply -f "$GITOPS_DIR/$APP/$APP.yaml" --wait >/dev/null 2>&1
   argocd app wait pipelines-service tektoncd --timeout=90 >/dev/null 2>&1
+  echo "OK"
+}
 
+postcheck() {
   # Wait until CRDs are synced to KCP
-  while [ "$(KUBECONFIG="$KUBECONFIG_KCP" kubectl api-resources | grep -c tekton.dev)" != 8 ]
+  echo -n "  - Sync CRDs to KCP: "
+  local length
+  length=$(echo $TEKTON_CRDS | tr ',' '\n' | wc -l)
+  local resources_regexp
+  resources_regexp=$(echo $TEKTON_CRDS | tr ',' '|')
+  local  wait_period=0
+  while [[ "$(KUBECONFIG="$KUBECONFIG_KCP" kubectl api-resources -o name  2>&1 | grep -E "$resources_regexp" | wc -l )" -ne $length ]]
```

Comment:
While this works, I think it can be a bit simpler :)
cr_tosync=$(echo $TEKTON_CRDS | tr ',' '\n' | wc -l)
cr_synced=$(echo KUBECONFIG="$KUBECONFIG_KCP" kubectl api-resources | grep -i "tekton" | wc -l)

while [[ $cr_tosync != $cr_synced  ]]

Reply:
The loop is not reeavluating cr_synced, so that would be an issue.